### ISSUE TITLE
[Bug]: Vue 2.7 usage - runtime errors + `legacy` seems to be removed but not mentioned in migration guide

### ISSUE THREAD

**Issue by jacekkarczmarczyk** [apertureless/vue-chartjs#966]
### Would you like to work on a fix?

- [ ] Check this if you would like to implement a PR, we are more than happy to help you go through the process.

### Current and expected behavior

Although it's mentioned in migration guide that the package supports Vue 2.7 I have difficulties with making it working.
First - importing from `vue-chartjs/legacy` doesn't work any more:

```js
// Module not found: Error: Package path ./legacy is not exported from package ...
import { Bar as BarChart } from 'vue-chartjs/legacy'; 
```

Second - importing from 'vue-chartjs' throws missing property errors, even after changing them from `chart-data` to `data` etc.

![image](https://user-images.githubusercontent.com/15625235/205892892-5caafe48-2eae-43bd-a6f3-56663909aa0c.png)

### Reproduction

https://stackblitz.com/edit/github-tqxopy?file=src%2FApp.vue

### chart.js version

4.0.1

### vue-chartjs version

5.0.0

**Comment by apertureless** [comment]
The migration guide is for vue-chartjs 3 -> 4
If you want to use vue-chartjs with vue 2.7 you have to lock your version to 4.x

As stated in the release notes and changelog we dropped the legacy support with vue-chartjs 5

https://github.com/apertureless/vue-chartjs/releases/tag/v5.0.0

**Comment by dangreen** [comment]
@apertureless Actually, it should work with vue 2.7 cause it supports composition API. I'm researching it right now.

**Comment by jacekkarczmarczyk** [comment]
@apertureless do you mean that https://vue-chartjs.org/migration-guides/#migration-from-v4-to-v5 is the migration guide 3 -> 4?

also I don't see dropping support for Vue 2.x in the release notes
![Uploading image.png…]()

**Comment by apertureless** [comment]
Sorry misread. I thought you're talking about vue < 2.7

**Comment by dangreen** [comment]
@jacekkarczmarczyk Released in [v5.0.1](https://github.com/apertureless/vue-chartjs/releases/tag/v5.0.1)

**Comment by jacekkarczmarczyk** [comment]
I can confirm it works fine now, but I'd suggest to add to the release notes and migration guide information that for Vue 2.7 you need to remove `/legacy` from the import path
